docs/tutorials/configure/pet-photographer.md

```diff
@@ -0,0 +1,984 @@
+1. Then, click **Save config**.
+
+![A color filter module that has been added.](/tutorials/pet-photographer/add-colorfilter-module.png)
+
+## Add services
+
+Add a [vision service](/services/vision/detection/) for color detection and a [data management service](/services/data/) for storing your filtered images.
+Consider the specific services your filter might require when working with other components.
+For example, the [sensors service](/services/sensors/) provides a central interface to all of your robot’s sensors.
+
+### Add data management service
+
+To enable data capture on your robot, add and configure the [data management service](/services/data/) to capture and store data on your robot's computer:
+
+{{< tabs >}}
+{{% tab name="Builder" %}}
+
+1. On the **Config** page, click the **Services** subtab and click **Create service** in the lower-left corner.
+1. Choose `Data Management` as the type and name your instance of the data manager `dm`.
+   This service syncs data from your robot to the Viam app in the cloud.
+1. Select **Create**.
+1. On the panel that appears, you can manage the capturing and syncing functions individually.
+   By default, the data management service captures data every 0.1 minutes in the <file>~/.viam/capture</file> directory.
+
+   You can leave the default settings as they are.
+   Click **Save config** at the bottom of the window.
+
+   ![An instance of the data management service named "dm". The cloud sync and capturing options are toggled on and the directory is empty. The interval is set to 0.1](/tutorials/pet-photographer/data-management-services.png)
+
+   For more detailed information, see [Add the data management service](/services/data/configure-data-capture/#add-the-data-management-service).
+   {{% /tab %}}
+   {{% tab name="JSON Template" %}}
+   Add the vision service object to the services array in your rover’s raw JSON configuration:
+
+```json {class="line-numbers linkable-line-numbers"}
+    {
+      "name": "dm",
+      "type": "data_manager",
+      "namespace": "rdk",
+      "attributes": {
+        "sync_interval_mins": 0.1,
+        "capture_dir": "",
+        "tags": [],
+        "additional_sync_paths": []
+      }
+    },
+  ... // Vision and other services
+```
+
+Click **Save Config** and head back to **Builder** mode.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Vision service to detect color
+
+This tutorial uses the color of my dog's collar, `#43A1D0` or `rgb(67, 161, 208)` (blue), but you can use a different color that matches your pet or a distinctly colored item on your pet.
+
+**Hex color #43A1D0**: {{<imgproc src="/tutorials/pet-photographer/43a1d0.png" resize="90x" declaredimensions=true alt="A color swatch for the color of the example subject's collar">}}
+
+To configure your [vision service color detector](/services/vision/detection/):
```

Comment:
Nit: let's introduce this in the same way we introduced the data management service:

```suggestion
To enable your smart machine to detect a specific color in its camera stream, add a [vision service color detector](/services/vision/detection/).
For this tutorial, we will configure the vision service to recognize a blue dog collar using `#43A1D0` or `rgb(67, 161, 208)`.
If you have a different item you want to use, or want to match to a color that matches your pet closely, you can use a different color.
```